grpc-gcp/src/main/java/com/google/grpc/gcp/GcpManagedChannelOptions.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.grpc.gcp;
+
+import io.opencensus.metrics.LabelKey;
```

Comment:
Curious why we want to use io.opencensus.metrics for the metrics?
Can you add some example usages to the comments like what gcp managedchannel metrics we can get and how to configure this option?
As someone who is not familiar with io.opencensus.metrics I feel the API is little bit confusing..

Reply:
Makes sense. Will do.

Reply:
Added an example of usage to the withMetricsOptions method.
Re: why opencensus: We need to provide some interface to emit metrics to and we can use the open sourced opencensus interface that many have integrations with instead of coming up with our own new interface. I know opencensus metrics is kinda deprecated and will be migrated to OpenTelemetry. But OpenTelemetry metrics is in alpha, so we will migrate when it will be ready. I think there will be a straightforward path to migrate from OpenCensus to OpenTelemetry for metrics.